src/main/java/Config.java

```diff
@@ -26,6 +32,7 @@ public Config getState() {
     @Override
     public void loadState(@NotNull Config config) {
         this.url = config.url;
+        this.defaultBranch = config.defaultBranch;
```

Comment:
what happens if the config is missing defaultBranch? I think that there should be a default value

Reply:
True. I will update the code, thanks for checking @muratsu 🙇

Reply:
@muratsu Updated here: https://github.com/sourcegraph/sourcegraph-jetbrains/pull/20/commits/b94653b82f1d5aadbea26c2809a2209ad77b8f9e

Reply:
@muratsu updated to set default value here https://github.com/sourcegraph/sourcegraph-jetbrains/pull/20/commits/db60dabf0705692d4c878b626186dce8354f3f46